tests/bgp/test_route_aggregation.py

```diff
@@ -0,0 +1,187 @@
+    num_v6_routes = gather_info['duthost'].shell('show ipv6 bgp neighbors {} advertised-routes | grep -c "*>"'.format(
+        gather_info['neigh_ip_v6']))['stdout']
+
+    # Configure and Advertise Loopback Networks on DUT
+    gather_info['duthost'].shell("sudo config loopback add Loopback11")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback11 1.1.1.1/24")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback11 1:1:1:1::/64")
+    gather_info['duthost'].shell("sudo config loopback add Loopback12")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback12 1.1.2.1/24")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback12 1:1:1:2::/64")
+    gather_info['duthost'].shell("sudo config loopback add Loopback13")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback13 1.1.3.1/24")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback13 1:1:1:3::/64")
+    gather_info['duthost'].shell("sudo config loopback add Loopback14")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback14 1.1.4.1/24")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback14 1:1:1:4::/64")
+    gather_info['duthost'].shell("sudo config loopback add Loopback15")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback15 1.1.5.1/24")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback15 1:1:1:5::/64")
+
+    cmd = 'vtysh -c "config" -c "route-map AGG_TEST_1 permit 10" -c "set as-path prepend 11111" -c "exit" -c "end"'
+    logger.debug(gather_info['duthost'].shell(cmd, module_ignore_errors=True))
+    cmd = 'vtysh -c "config" -c "route-map AGG_TEST_2 permit 10" -c "set as-path prepend 22222" -c "exit" -c "end"'
+    logger.debug(gather_info['duthost'].shell(cmd, module_ignore_errors=True))
+    cmd = 'vtysh{} -c "config" -c "router bgp {}" -c "address-family ipv4 unicast" \
+          -c "network 1.1.1.0/24 route-map AGG_TEST_1" -c "network 1.1.2.0/24 route-map AGG_TEST_1" \
+          -c "network 1.1.3.0/24 route-map AGG_TEST_2" -c "network 1.1.4.0/24 route-map AGG_TEST_2" \
+          -c "network 1.1.5.0/24" -c "address-family ipv6 unicast" -c "network 1:1:1:1::/64 route-map AGG_TEST_1" \
+          -c "network 1:1:1:2::/64 route-map AGG_TEST_1" -c "network 1:1:1:3::/64 route-map AGG_TEST_2" \
+          -c "network 1:1:1:4::/64 route-map AGG_TEST_2" -c "network 1:1:1:5::/64 route-map AGG_TEST_2"'.format(
+              gather_info['cli_options'], gather_info['dut_asn'])
+    logger.debug(gather_info['duthost'].shell(cmd, module_ignore_errors=True))
+    verify_route_agg(gather_info['duthost'], gather_info['neigh_ip_v4'], gather_info['neigh_ip_v6'], "5", False)
+    check_baseline(gather_info['duthost'], gather_info['neigh_ip_v4'], gather_info['neigh_ip_v6'], num_v4_routes,
+                   num_v6_routes)
+
+    # Configure summary-only route aggregation
```

Comment:
same here, seems repeated code, could we encapsulate this into a function?

Reply:
These config changes are not repeated anywhere.